src/explore-education-statistics-admin/src/config/ckEditorConfig.ts

```diff
@@ -74,7 +72,7 @@ export const pluginsConfigLinksOnly: ReadonlySet<PluginName> =
   new Set<PluginName>(['Link']);
 
 export const pluginsConfigSimple: ReadonlySet<PluginName> = new Set<PluginName>(
-  ['Bold', 'Italic', 'Link', 'List'],
+  ['Bold', 'Link', 'List'],
```

Comment:
As well as removing it here it can be removed from the PluginName and ToolbarOption types in  https://github.com/dfe-analytical-services/explore-education-statistics/blob/dev/src/explore-education-statistics-admin/src/types/ckeditor.ts

Reply:
Ahh, good spot! I've now removed those